app-nexus-network/app-nexus-network-htb.js

```diff
@@ -184,6 +184,9 @@ function AppNexusNetworkHtb(configs) {
                 consent_required: gdprStatus.applies,
                 consent_string: gdprStatus.consentString
             };
+
+            var uspStatus = ComplianceService.usp.getConsent();
```

Comment:
Just a FYI. If the CCPA times out or the API is down, we will return an empty string for uspString.